browser/js/common/directives/navbar/navbar.html

```diff
@@ -1,30 +1,78 @@
-<nav class="navbar navbar-static-top">
-    <div class="container">
-        <ul>
-            <li>
-                <a ng-click="goToSection('.problem')"> The Problem </a>
-            </li>
+<nav id='navbar' class='navbar navbar-static-top'>
+	<div class='container'>
+		<div class='dropdown'>
+			<div class='dropdown-title'>
+				What is Kinetic
+			</div>
+				<div class='dropdown-content'>
+					<div class='list-container'>
+						<a ng-click="goToSection('.problem', 'home')"> The Problem </a>
+						<a ng-click="goToSection('.solution', 'home')"> Our Solution </a>
+						<a ng-click="goToSection('.what-is', 'home')"> What We Do </a>
+						<a ng-click="goToSection('.about-us', 'home')"> About Us </a>
+						<a ng-click="goToSection('.sponsored', 'home')"> Key Issues </a>
+						<a ng-click="goToSection('.get-involved', 'home')"> Get Involved </a>
+					</div>
+				</div>
+		</div>
```

Comment:
yeah, so right now there's a div with the class `list-container`. i'd suggest throwing a `data-state="home"`. when an `a` is clicked, it will look at its containing div's `data-state` to get which state to go to and look at its own `data-element` (or something else like that) to get which element to scroll to.